modules/java_api/cpp/compatibility/blob.cpp

```diff
@@ -1,11 +1,11 @@
 #include <inference_engine.hpp>
 
-#include "openvino_java.hpp"
-#include "jni_common.hpp"
+#include "../openvino_java.hpp"
+#include "../jni_common.hpp"
```

Comment:
Don't use relative paths in include - define `target_include_directories` in CMake instead